samples/tfm_integration/psa_firmware/src/shell.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019,2020 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <ctype.h>
+#include <shell/shell.h>
+
+#if CONFIG_PSA_SHELL
+
+static int
+psa_shell_invalid_arg(const struct shell *shell, char *arg_name)
+{
+	shell_print(shell, "Error: invalid argument \"%s\"\n", arg_name);
+
+	return -EINVAL;
+}
+
+static int
+psa_shell_cmd_version(const struct shell *shell, size_t argc, char **argv)
+{
+	shell_print(shell, "%s", "0.0.0");
```

Comment:
Please update this to display the current firmware version(s).

Reply:
Done

Reply:
Can we also display the S and MCUBoot version numbers? I don't know if MCUBoot is available, but at least S is useful to know.